js/libs/builder.js

```diff
@@ -800,6 +800,8 @@ var app = new Vue({
       if (value === 'settings') {
         $vm.setupCodeEditor();
         changeSelectText();
+      } else {
+        tinymce.remove();
```

Comment:
Wouldn't it be better to basically only initialise TinyMCE once? Let me know if you agree or removing it is cleaner based on the rest of the code.

Reply:
I tried to initialize TinyMCE once but in that case bug still occurs.